vagrant/provisioning/roles/upgrade-solr/tasks/main.yml

```diff
@@ -0,0 +1,65 @@
+- name: stop arkcase
+  become: yes
+  systemd:
+    daemon_reload: true
+    name: arkcase
+    state: stopped
+
+- name: copy existing solr data to backup folder
+  become: yes
+  copy:
+    remote_src: yes
+    src: "{{ root_folder }}/data/solr/"
+    dest: "{{ root_folder }}/data/solr_backup/"
+    owner: solr
+    group: solr
+
+- name: delete solr cores from previous install
+  command: /opt/arkcase/app/solr/bin/solr delete -c {{ item }}
+  loop: "{{ solr_cores }}"
+
+- name: stop solr
+  become: yes
+  systemd:
+    daemon_reload: true
+    name: solr
+    state: stopped
+
+- name: Remove solr config files and folders
+  become: yes
+  file:
+    path: "{{ item }}"
+    state: absent
+    group: solr
+    owner: solr
+  loop:
+    - "{{ root_folder }}/data/solr"
+    - "{{ root_folder }}/app/solr"
+    - "{{ root_folder }}/install/solr"
+    - "/etc/systemd/system/solr.service"
+
+- name: install solr
+  include_role: 
+    name: solr
+
+- name: remove solr default data folder after upgrade
+  become: yes 
+  shell: rm -rf {{ root_folder }}/data/solr/*
+
+- name: copy backup dir to new solr data dir
+  become: yes
```

Comment:
Could this be a move instead of a copy?

Reply:
Well, it could be, but this is brand new data directory, so no backed up data here, just what do we get after installation. So, no real point of backing up on that. Or, maybe i am missing something...